engine/src/game_config.h

```diff
@@ -19,28 +19,48 @@ using pt::ptree;
 
 
 class GameConfig
-{ 
+{
 private:
     static std::map<string, string> variables;
-    static string _GetVariable(string const &section, string const &name);
-
+    static string DEFAULT_ERROR_VALUE;
+    static inline string _GetVariable(string const &section, string const &name)
+    {
+        string const key = section + "." + name;
+        if (variables.count(key))
+            return variables[key];
+        return DEFAULT_ERROR_VALUE;
+    }
 public:
     static void LoadGameConfig(const string &filename);
     template <class T>
-    static T GetVariable(string const &section, string const &name, T default_value) = delete;
+    static inline T GetVariable(string const &section, string const &name, T default_value) = delete;
```

Comment:
Xenial from Docker has an older version of clang. 3.8, I believe. clang 3.8 on Xenial apparently doesn't handle well `= delete` template methods with specializations. See https://stackoverflow.com/a/33258249/5067822 . For this reason, I have moved `Xenial clang` to Allowed Failures on Travis.

Reply:
@stephengtuggy does it help to do a full upgrade on the docker image:
```
# apt-get update
# apt-get -y upgrade
# apt-get -y dist-upgrade
```

FYI - created https://hub.docker.com/orgs/vegastrike for you